src/pickup.cpp

```diff
@@ -848,12 +848,10 @@ void Pickup::pick_up( const tripoint &p, int min, from_where get_items_from )
                     std::string item_name;
                     std::string stolen;
                     bool stealing = false;
-                    if( this_item.has_owner() ) {
-                        const faction *item_fac = this_item.get_owner();
-                        if( item_fac != g->u.get_faction() ) {
-                            stolen = "<color_light_red>!</color>";
-                            stealing = true;
-                        }
+                    if( this_item.has_owner() &&
+                        this_item.get_owner() != g->faction_manager_ptr->get( faction_id( "your_followers" ) ) ) {
```

Comment:
What's wrong with using `g->u.get_faction()` here?

Reply:
Ah nothing, I thought get_faction() might be what was causing the problem and forgot to put the code back to how it was.